src/index.css

```diff
@@ -27,7 +27,7 @@ input {
 
 .light {
   --background-color: #eaedf0;
-  --text-color-first: #000000;
+  --text-color-first: rgb(43, 43, 43);
```

Comment:
@SubhamKarmakar24 , `rgb(43, 43, 43)` is something I added. In my opinion we can keep the legacy setting ( i.e. `#000000`).
Basically, we can keep changes in this file untouched.
Code changes in `src/components/Results.css` are good enough.

Reply:
Yes, fixed it through #32